python/lsst/meas/astrom/matchPessimisticB.py

```diff
@@ -402,8 +402,10 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources,
         # the consensus mode of the matcher. If not default to Optimistic
         # pattern matcher behavior.
         numConsensus = self.config.numPatternConsensus
-        if ref_array.shape[0] < self.config.numBrightStars or \
-           src_array.shape[0] < self.config.numBrightStars:
+        if ref_array.shape[0] < self.config.numBrightStars + \
+           self.config.numPointsForShapeAttempt or \
+           src_array.shape[0] < self.config.numBrightStars + \
+           self.config.numPointsForShapeAttempt:
```

Comment:
I suggest an intermediate variable since it is a simple sum of two config variables. The only reason to put it in the config would be if the user might want to use some other value, but unless that's likely it just adds clutter.